kernel/linux-container.spec-template

```diff
@@ -0,0 +1,127 @@
+#
+# This is a special configuration of the Linux kernel, aimed exclusively
+# for running inside a container
+# This specialization allows us to optimize memory footprint and boot time.
+#
+
+%define bzimage_arch x86
+
+Name:           linux-container
+Version:        @VERSION@
+Release:        @RELEASE@.<B_CNT>
```

Comment:
This is 91 in 'release' repo is this ok?
https://build.opensuse.org/package/view_file/home:katacontainers:release/linux-container/linux-container.spec?expand=1